### PR TITLE
Add disclaimer about standalone swarm in 1.12+

### LINKED ISSUE #559: Feedback for: swarm/index.md
File: [swarm/index.md](https://docs.docker.com/swarm/)

Please add a warning or disclaimer indicating that this is not the latest or preferred method of managing a swarm, considering this is the first result on a google search for "Docker swarm"

### PR DESCRIPTION
**Pull request by mstanleyjones**
### Describe the proposed changes

Add disclaimer about standalone swarm vs swarm mode in 1.12+.

### Unreleased project version

n/a

### Related issue

Fixes #559 

### Related issue or PR in another project

n/a

### Please take a look

@aaronlehmann @fresnizky

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit ade939b755f603028497e5f686bf1c8c2186829d

https://deploy-preview-578--docker-docs.netlify.com

File: swarm/index.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ keywords: docker, swarm,  clustering
 title: Docker Swarm
 ---
 
+This section refers to standalone Docker Swarm. If you use Docker 1.12 or higher,
+[Swarm mode](../engine/swarm/index.md) is integrated with Docker Engine, and you
+should use Swarm mode instead.
```
Comment (mstanleyjones) [inline swarm/index.md]: If there are things you can't do with the builtin version we should probably enumerate them. I'm open to suggestions.

Comment (mgoelzer) [inline swarm/index.md]: @aaronlehmann I'm fine with your wording.  I think our position is that 1.12 services is recommended over `docker/swarm`

@mstanleyjones The older `docker/swarm` is simpler so it's better for some setups like Jenkins slaves.  But it's not a specific enumerable set of features, it's more about a simpler model.

**Comment by mstanleyjones** [comment]
@aaronlehmann @mgoelzer thanks for your suggestions. PTAL again and let me know if this is better. Thanks!

File: swarm/index.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ keywords: docker, swarm,  clustering
 title: Docker Swarm
 ---
 
+This section refers to standalone Docker Swarm. If you use Docker 1.12 or higher,
+[Swarm mode](../engine/swarm/index.md) is integrated with Docker Engine. Most
+users should use the integrated Swarm mode. Standalone Docker Swarm is not
+integrated into the Docker Engine API and CLI commands. However, it is suitable
+for some lightweight use cases because it is simpler to set up.
```
Comment (aaronlehmann) [inline swarm/index.md]: Let's remove the last sentence :)

Comment (mstanleyjones) [inline swarm/index.md]: Done.

File: swarm/index.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ keywords: docker, swarm,  clustering
 title: Docker Swarm
 ---
 
+This section refers to standalone Docker Swarm. If you use Docker 1.12 or higher,
+[Swarm mode](../engine/swarm/index.md) is integrated with Docker Engine. Most
+users should use the integrated Swarm mode. Standalone Docker Swarm is not
+integrated into the Docker Engine API and CLI commands..
```
Comment (mstanleyjones) [inline swarm/index.md]: Fixed in master, thanks.

**Pull request merged**